day_two_guide/topics/project_backup.adoc

```diff
@@ -202,7 +202,7 @@ $ less bcs.json
 +
 [NOTE]
 ====
-Some files can be empty if the objects does not exist when the export is
+Some files can be empty if the object does not exist when the export is
```

Comment:
Maybe "If objects do not exist during export, some files are empty."

If you have time/inclination, "execute" isn't the best word to use. I'd check and see if you can replace them with variants of "export."

Reply:
Changed to "If the original object does not exist, empty files will be created when exporting."